pkg/subctl/cmd/deploybroker.go

```diff
@@ -28,12 +28,15 @@ import (
 
 var enableDataplane bool
 var disableDataplane bool
+var enableLighthouse bool
```

Comment:
Since the user-facing option is generic, I’d recommend keeping the variable generic too:

    var enableServiceDiscovery bool

Reply:
Was in two minds, decided to go this way to get inputs in review.